salt/daemons/masterapi.py

```diff
@@ -609,13 +609,17 @@ def _mine_get(self, load, skip_verify=False):
         minions = _res['minions']
         for minion in minions:
             fdata = self.cache.fetch('minions/{0}'.format(minion), 'mine')
-            if isinstance(fdata, dict):
+
+            if fdata is None or not isinstance(fdata, dict):
```

Comment:
I don't think you need the first comparison with None, since that's not an instance of dict.

Doing: if not isinstance(fdata, dict) should suffice